src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -440,7 +440,7 @@ private async Task<NegotiationResponse> NegotiateAsync(Uri url, HttpClient httpC
                 // Get a connection ID from the server
                 Log.EstablishingConnection(logger, url);
                 var urlBuilder = new UriBuilder(url);
-                if (!urlBuilder.Path.EndsWith("/"))
+                if (!urlBuilder.Path.EndsWith("/", StringComparison.Ordinal))
```

Comment:
Can this be changed to a character instead of string?

Reply:
This one cross-compiles to ns2.0 which does not have the char overload.